src/vivarium/framework/components/manager.py

```diff
@@ -16,47 +19,63 @@ class ComponentConfigError(VivariumError):
 
 
 class ComponentManager:
-    """Handles the setup and access patterns for components in the system."""
+    """Handles tracking and access patterns for all components in a Vivarium
```

Comment:
We have a mix of Vivarium and ``vivarium`` when refering to the framework.  I'm leaning towards the latter, and have been writing it as such in all my recent docs.  We should be consistent about it.

Also, try to give docstrings of the format:

```
"""Single line description.

More detailed description across
multiple lines.

Parameters
---------------
...
etc.
            <- Final newline before end of comment.
"""
```